### DISCUSSION TITLE
Selecting multiple cells to copy

### DISCUSSION THREAD

**Discussion by jenfenghsu** [fonsp/Pluto.jl#880] [Questions about using Pluto] [answered]
Is there a way to select multiple cell for moving or copying?  I have tried naïve ways like Shift+click, Ctrl+click, and click and drag over multiple cells, but wasn't lucky enough.  I realize that a better habit to develop is to fight my inertia and write functions to reuse lines of codes, so I don't have to keep copying/pasting large amount of codes.

**Comment by icweaver** [comment, marked as answer]
Hey @jenfenghsu! Just doing the typical click and drag starting from the right hand side to select cells, then click and drag on the left hand side to move cells or ctrl-c ctrl-v to copy/paste seems to work for me. This is on v0.12.20

**Reply by jenfenghsu** [reply]
Ah great!  Manipulating from the right hand side was the key that I missed.  Thanks very much.  Mine was v0.12.19 and it worked well, too.

**Reply by Realife-Brahmin** [reply]
> Hey @jenfenghsu! Just doing the typical click and drag starting from the right hand side to select cells, then click and drag on the left hand side to move cells or ctrl-c ctrl-v to copy/paste seems to work for me. This is on v0.12.20

Drag select (from the right side of the cells) you meant. Sorry I didn't immediately understand your first sentence. 😅
Also, is there a method to select cells which are not continuous?
